composer.json

```diff
@@ -28,7 +28,8 @@
         "symfony/dependency-injection": "2.8.2",
         "typhonius/behattask": "1.0",
         "drupal/state_machine": "~8.1.0-beta2",
-        "drupal/ds": "^8.2"
+        "drupal/ds": "^8.2",
+        "drupal/inline_entity_form": "8.1.x-dev"
```

Comment:
It better satisfies my inner OCD if these are maintained in alphabetical order :)